redfish-core/lib/update_service.hpp

```diff
@@ -60,6 +61,91 @@ inline static void activateImage(const std::string& objPath,
             "Active"));
 }
 
+inline void updateErrorLogMessage(
+    const std::shared_ptr<task::TaskData>& taskData,
+    const std::string& index)
+{
+    using AdditionalDataType = std::vector<std::string>;
+    using GetManagedPropertyType = boost::container::flat_map<
+        std::string,
+        std::variant<std::string, AdditionalDataType>>;
+    using GetManagedObjectsType = boost::container::flat_map<
```

Comment:
We moved to common D-Bus types upstream, can you use  #include <dbus_utility.hpp>

Reply:
E.g. 
https://github.com/openbmc/bmcweb/blob/3d768a165183dd8cf7e485ac67ab7270845d5f87/redfish-core/lib/certificate_service.hpp#L868